src/renderer/App.scss

```diff
@@ -27,11 +27,11 @@ body {
 }
 
 .dex-entry {
-  width: 370px;
-  textarea{
-  font-family: 'Pokemon', monospace;
-  white-space: nowrap;
-}
+  width: 410px;
```

Comment:
not sure if this was only an issue on my computer or not, but the width here wasn't enough with the monospaced font to fit 40 characters per line.

Reply:
Have to play with this number, the font isn't monospaced here or in game. Trying to give a good approximation of what will fit, but ultimately we'll have to tweak some entries based on what we see in game